apikey-service/pom.xml

```diff
@@ -71,6 +71,12 @@
             <version>${spring-boot.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20140107</version>
```

Comment:
Why use such an old version and not the latest: https://mvnrepository.com/artifact/org.json/json/20190722?